content/en/docs/Writing policies/external-data-sources.md

```diff
@@ -0,0 +1,440 @@
+---
+title: External Data Sources
```

Comment:
Now that I see this, "External Data Sources" (which the current docs state as well) is somewhat, I think, misleading. It's external in the sense it doesn't come from AdmissionReview, but not external to Kubernetes. How would we then refer to truly external data sources when, for example, JS is implemented and users actually reach outside to pull in data? That's truly external at that point. Is this maybe better called "Cluster Data Sources" or "API Data Sources"?

Reply:
"Intra-Cluster Data Sources" ?
"Kubernetes Data Sources" ?

Reply:
I see your point, buts its all relative. I was thinking external as in "external data" from any app / service independent of in-cluster or out of cluster. For example, we do not allow making calls to other REST APIs in-cluster.

We could generalize to "Data Sources" and drop "external".

Reply:
Yeah, I understand what you mean. I don't think it's a big deal either way and the explanation is clear when a user reads the text. It's just I think that the typical user who reads "external" in the context of Kubernetes is automatically thinking "something outside my cluster boundary."